addons/crm/models/crm_lead.py

```diff
@@ -870,34 +870,6 @@ def opps_key(opportunity):
 
         return self.sorted(key=opps_key, reverse=reverse)
 
-    def get_duplicated_leads(self, partner_id, include_lost=False):
```

Comment:
Maybe you can add in commit message that this method was only used in enterprise module (to explain this is why we cannot find any usage of it in this commit)